src/course/en/components.json

```diff
@@ -68,13 +80,14 @@
         "body": "This is the graphic component body. You can introduce your graphic or you may just want to let the graphics do the talking.<br><br>Component can either be single or spanned.",
         "instruction": "",
         "_graphic": {
-            "large": "course/en/images/full_width.jpg",
-            "small": "course/en/images/full_width.jpg",
+            "large": "course/en/images/full-width.png",

```

Comment:
could we change `"large"` to `"src"` to show how you can avoid specifying both when you don't want different images for large and small?